contents/blog/the-posthog-array-1-34-0.md

```diff
@@ -0,0 +1,111 @@
+---
+date: 2022-03-24
+title: Array 1.34.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes", "Product updates"]
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: 
+---
+
+PostHog 1.34.0 introduces...(Excerpt info here)
+
+<blockquote class='warning-note'>
+<b>IMPORTANT!</b> Before upgrading, make sure to run all async migrations on your instance. This version will not run until async migrations are completed. Please check out the <a href="/docs/self-host/configure/async-migrations/overview" target="_blank">async migrations</a> docs for details.
+</blockquote>
+
+## PostHog 1.34.0 release notes
+
+> Wondering how to upgrade a self-hosted instance? Check out our [upgrade guide](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Homepage](#new-homepage)
+- [Data Management](#new-data-management)
+- [Property Group Filtering](#new-property-group-filtering)
+- [Smart Insight Naming](#improvement-smart-insight-naming)
+- [Staff Users](#new-staff-users)
+
+### New: Homepage
```

Comment:
@rcmarron can you write a brief summary of the homepage work for this release? :D

Reply:
How about:

> Projects have a new homepage that help you view your most important insights as quick as possible. This homepage allows you to pin a customizable dashboard to it, so you can configure the metrics that you and your team sees each time you go PostHog.